RInAppMessaging/Classes/Extensions/UIView+IAM.swift

```diff
@@ -20,6 +20,18 @@ extension UIView {
     func removeAllSubviews() {
         subviews.forEach { $0.removeFromSuperview() }
     }
+
+    func findIAMViewSubview() -> UIView? {
+        for subview in subviews {
+            if subview is BaseView {
+                return subview
+            } else if let iamView = subview.findIAMViewSubview() {
+                return iamView
+            }
+        }
```

Comment:
I'm wondering if you can use subviews.first{}:
```
let result = subviews.first { (value) -> Bool in
             if subview is BaseView {
                 return true
             } else if let iamView = subview.findIAMViewSubview() {
                 return true
             }
}
return result
```

Reply:
Hmm, it's going to break from the loop on the first element that returns true... so yeah I think it should work 👍

Reply:
Ah not really, the nested `$0.findIAMViewSubview()` call returns UIView which I have to discard. Returning true in that case will end up `result` being a parent view